doc/AddASetting.md

```diff
@@ -1,40 +1,40 @@
-    * define the property key ID and the property key itself
-        - Yes, the large majority of the `DEFINE_PROPERTYKEY` defs are the same, it's only the last byte of the guid that changes
+* For now, you can update winconp.h with your consumable changes.
+* Define the registry name (e.g. `CONSOLE_REGISTRY_CURSORCOLOR`)
+* Add the setting to `CONSOLE_STATE_INFO`
+* Define the property key ID and the property key itself (Yes, the large majority of the `DEFINE_PROPERTYKEY` defs are the same, it's only the last byte of the guid that changes)
 
-2. Add matching fields to Settings.hpp
-    - add getters, setters, the whole drill.
+### 2. Add matching fields to Settings.hpp
+Add getters, setters, the whole drill.
 
-3. Add to the propsheet
-    - We need to add it to *reading and writing* the registry from the propsheet, and *reading* the link from the propsheet. Yes, that's weird, but the propsheet is smart enough to re-use ShortcutSerialization::s_SetLinkValues, but not smart enough to do the same with RegistrySerialization.
-    - `src/propsheet/registry.cpp`
-        -  `propsheet/registry.cpp@InitRegistryValues` should initialize the default value for the property.
-        -  `propsheet/registry.cpp@GetRegistryValues` should make sure to read the property from the registry
+### 3. Add to the propsheet
+We need to add it to *reading and writing* the registry from the propsheet, and *reading* the link from the propsheet. <br>
+Yes, that's weird, but the propsheet is smart enough to re-use ShortcutSerialization::s_SetLinkValues, but not smart enough to do the same with RegistrySerialization. <br>
+- `src/propsheet/registry.cpp`
+   -  `propsheet/registry.cpp@InitRegistryValues` should initialize the default value for the property.
+   -  `propsheet/registry.cpp@GetRegistryValues` should make sure to read the property from the registry
 
-4. Add the field to the propslib registry map
+### 4. Add the field to the propslib registry map
 
-5. Add the value to `ShortcutSerialization.cpp`
-    - Read the value in `ShortcutSerialization::s_PopulateV2Properties`
-    - Write the value in `ShortcutSerialization::s_SetLinkValues`
+### 5. Add the value to `ShortcutSerialization.cpp`
+- Read the value in `ShortcutSerialization::s_PopulateV2Properties`
+- Write the value in `ShortcutSerialization::s_SetLinkValues`
 
-6. Add the setting to `Menu::s_GetConsoleState`, and `Menu::s_PropertiesUpdate`
-Now, your new setting should be stored just like all the other properties.
+### 6. Add the setting to `Menu::s_GetConsoleState`, and `Menu::s_PropertiesUpdate`
+Now, your new setting should be stored, just like all the other properties.
 
-7. Update the feature test properties to get add the setting as well
-    - `ft_uia/Common/NativeMethods.cs@WinConP`:
-        - `Wtypes.PROPERTYKEY PKEY_Console_`
-        - `NT_CONSOLE_PROPS`
+### 7. Update the feature test properties to get add the setting as well
+`ft_uia/Common/NativeMethods.cs@WinConP`:

```

Comment:
This particular sub point seems weird now, it feels like it lost the connection to the parent bullet point (which is now the header)

Reply:
What do you suggest should be done?